File: Examples/test-suite/php/director_ignore_runme.php
<?php

require "tests.php";

// No new functions
check::functions(array());
check::classes(array('DAbstractIgnores','DIgnores','DTemplateAbstractIgnoresInt','DIgnoreConstructor','DIgnoreOnlyConstructor','DIgnoreDestructor'));
// No new vars
check::globals(array());

class DIgnoresDerived extends DIgnores {
  function PublicMethod1() {
    return 18.75;
  }
}

class DAbstractIgnoresDerived extends DAbstractIgnores {
}

$a = new DIgnoresDerived();
check::equal($a->Triple(5), 15, "Wrong return value");

$b = new DAbstractIgnoresDerived();
check::equal($b->Quadruple(5), 20, "Wrong return value");

File: Examples/test-suite/lua/director_ignore_runme.lua
require("import") -- the import fn
import("director_ignore") -- import lib

local a = director_ignore.DIgnores()
function PublicMethod1(self)
    return 18.75
end
swig_override(a, 'PublicMethod1', PublicMethod1)
assert(a:Triple(5) == 15)

local b = director_ignore.DAbstractIgnores()
assert(b:Quadruple(5) == 20)
